spec/models/garden_spec.rb

```diff
@@ -90,7 +90,7 @@
 
   context 'ordering' do
     it "should be sorted alphabetically" do
-      z = FactoryGirl.create(:garden_z)
+      FactoryGirl.create(:garden_z)
       a = FactoryGirl.create(:garden_a)
```

Comment:
This one might have been better fixed by fleshing out the other expectation (garden.last == z)